sympy/plotting/pygletplot/util.py

```diff
@@ -122,9 +122,9 @@ def interpolate(a_min, a_max, a_ratio):
 
 def rinterpolate(a_min, a_max, a_value):
     a_range = a_max - a_min
-    if a_range == 0:
+    if a_max == a_min:
         a_range = 1.0
-    return (a_value - a_min) / float(a_range)
+    return a_value - a_min / float(a_range)
```

Comment:
Stupid error which I will revert in the next push. Will see the status of the tests first though...